pages/common/dvc-diff.md

```diff
@@ -0,0 +1,24 @@
+# dvc diff
+
+> Show changes in DVC tracked file and directories added/deleted/modified.
+> More information: <https://dvc.org/doc/command-reference/diff>.
+
+- Compare DVC tracked file from different Git commit/tag/branch w.r.t current workspace:
+
+`dvc diff {{commit_hash/tag/branch}}`
+
+- Compare DVC tracked file in Git b_rev w.r.t a_rev:
```

Comment:
Perhaps this will read better?
```suggestion
- Compare the differences in DVC tracked files across 2 Git commits:
```

Reply:
The comparison is not commutative. i.e 
- `dvc diff b_rev a_rev` is not equal to `dvc diff a_rev b_rev`.

Thus deleted in one becomes added in other. I think we need to communicate this too.

Reply:
Ah, I see - good point. What  about this then?
```suggestion
- Compare the changes in DVC tracked files from 1 git commit to another:
```